hack/bazel-race.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+source hack/common.sh
+source hack/bootstrap.sh
+source hack/config.sh
+
+set -ex
+WHAT=${WHAT:-"//staging/src/kubevirt.io/...:* //pkg/...:* //cmd/...:* //tools/util/...:* //tools/cache/...:* //tests/framework/...:*"}
+bazel run --config=${ARCHITECTURE} -- :buildozer -types go_test 'set race "on"' ${WHAT}
+bazel run --config=${ARCHITECTURE} -- :buildozer -types go_test 'set race "off"' //pkg/virt-api/webhooks/fuzz/...:*
```

Comment:
@fossedihelm @0xFelix - I've rewritten this to use `buildozer` but the this does impact the test run time and it's still uncovering races AFAICT. Before I continue and clean this up do you think we should still enable this by default for our unit test runs or should I instead add another optional `test-race` target?

Reply:
We are already using `-race` on the s390x unit-tests, so a lot of the race conditions in tests are already fixed.

Reply:
> @lyarwood Thank you! Do you have any data on the extent of the impact?

~23mins (with virt-operator test timeout) run time versus the current 11mins.

https://prow.ci.kubevirt.io/job-history/gs/kubevirt-prow/pr-logs/directory/pull-kubevirt-unit-test

> We are already using `-race` on the s390x unit-tests, so a lot of the race conditions in tests are already fixed.

@jschintag I wonder why ca05b5650de548a234e8eae6047598c96079c676 and 5f16adccf672f47ff9fd260f71c3e0813ee5bca4 are required now?

Reply:
> ~23mins (with virt-operator test timeout) run time versus the current 11mins.

Ouch! @brianmcarey @dhiller Maybe it's better to have a separate lane? Maybe periodic? Any thoughts?